forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/services/store/tpp/TppStoreServiceImpl.java

```diff
@@ -79,6 +79,24 @@
         }
     }
 
+    @Override
+    public List<Tpp> findByOrganisationId(String orgId) {
+        UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl
+                (rsStoreRoot + "/tpps/search/findByOrganisationId");
+        builder.queryParam("organisationId", orgId);
+        URI uri = builder.build().encode().toUri();
+        LOGGER.debug("Find organistationId {}", orgId);
+        try {
+            return new ArrayList<>(restTemplate.exchange(uri, HttpMethod.GET, null,
+                    new ParameterizedTypeReference<Resources<Tpp>>() {}).getBody().getContent());
+        } catch (HttpClientErrorException e) {
+            if (e.getStatusCode() == HttpStatus.NOT_FOUND) {
```

Comment:
Need to make sure this gets caught and translated to the correct OAuth2 type exception for return to client if relevant.